docs/mc_definition.tex

```diff
@@ -770,10 +770,22 @@ \subsubsection{hera\_autos}
 \textbf{antenna\_number} & int & antenna number \\ \hline
 \textbf{antenna\_feed\_pol} & string & antenna feed polarization, either `e' or `n'. \\ \hline
 measurement\_type* & string & Currently can only be `median'. \\ \hline
-value* & float & Measured value. \\ \hline
+value* & float & calculated value (median across frequencies). \\ \hline
 \end{tabular}
 \end{center}
 
+\subsubsection{hera\_auto\_spectrum}
```

Comment:
I see this uses the same subsubsection heading as the hera_autos, but I don't see a `hera_auto_spectrum` line in the table of contents for section 4 in the PDF. Can you verify it's absence or presence on your end?

Reply:
good catch!! I needed to compile one more time to make all the references work their way through.